content/week-two/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Week Two
+content_type: topic
+submission type:  
+---
+
+# Challenge: Improve working professionals' experiences of working remotely
+
+### The Brief
+
+**Context** 

```

Comment:
I'd also remove **context**. If the word context is necessary then it would be better to sort out the heading that says The Brief so that it says something more useful. 

It's just a preference though. It just seems untidy to me. 

If you want to keep that word there then you should use hash symbols to show that it is a heading. `#### Context`. Reason being: when this thing gets turned into a web page headings are drawn with specific html elements.  At some point you might want to make all your headings looks a certain way, eg they all use the same font.  If you use **bold**  when you mean heading then the final result will look inconsiostent